src/components/Tool/FixEmailAddresses/FixEmailAddresses.tsx

```diff
@@ -360,20 +370,47 @@ export const FixEmailAddresses: React.FC<FixEmailAddressesProps> = ({
           </Grid>
           {!!data.people.nodes.length ? (
             <>
-              <Grid item xs={12}>
-                {data?.people.nodes.map((person) => (
-                  <FixEmailAddressPerson
-                    person={person}
-                    key={person.id}
-                    dataState={dataState}
-                    accountListId={accountListId}
-                    handleChange={handleChange}
-                    handleChangePrimary={handleChangePrimary}
-                    handleSingleConfirm={handleSingleConfirm}
-                    setContactFocus={setContactFocus}
-                  />
-                ))}
-              </Grid>
+              <InfiniteList
+                loading={loading}
+                data={data.people.nodes}
+                itemContent={(index, person) => (
+                  <Grid
+                    key={index}
+                    item
+                    xs={12}
+                    sx={{
+                      marginBottom: `${theme.spacing(2)}`,
+                    }}
+                  >
+                    <FixEmailAddressPerson
+                      person={person}
+                      key={person.id}
+                      dataState={dataState}
+                      accountListId={accountListId}
+                      handleChange={handleChange}
+                      handleChangePrimary={handleChangePrimary}
+                      handleSingleConfirm={handleSingleConfirm}
+                      setContactFocus={setContactFocus}
+                    />
+                  </Grid>
+                )}
+                endReached={() =>
+                  data.people.pageInfo.hasNextPage &&
+                  fetchMore({
+                    variables: { after: data.people.pageInfo.endCursor },
+                  })
+                }
+                EmptyPlaceholder={<NoData tool="fixEmailAddresses" />}
+                style={{
```

Comment:
I'm seeing a tiny amount of double scrolling. I think this calc may need to be adjusted to make the list slightly smaller.

https://github.com/user-attachments/assets/275e954c-a9d6-4a41-a5f3-0e57861bd4a1

Reply:
Changing that value can increase the double scroll but not get rid of it entirely it seems. If I hide the scrolling on `overflow-y` on the `<div class="css-1wkgudm">` I can get rid of the extra scroll bar. However, no height changes I've tried in the DOM make it go away.

Reply:
Double scrolling issues are the bane of my existence. I can't even reproduce it anymore, so I think this is OK as-is.